canonical-web-and-design/ubuntu.com
# New page /robotics/esm

## Done

- New page /robotics/esm

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/robotics/esm
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1oiz_006n9U1Ya50wdSkKIyj_jsxSr92FdCpkDU4kpHs/edit?ts=606301c9#)


## Issue / Card

Fixes [#4001](https://github.com/canonical-web-and-design/web-squad/issues/4001)
